### PR TITLE
Admin Deletion GUI

### LINKED ISSUE #139: Add Admin GUI to remove products
### Pending:
- [x] #124
- [x] #138 

### Requirements:
- [x] New page protected by AdminRoute
- [x] Display all products in database
- [x] Admin user should be able to delete products
- [x] On deletion, product is removed from database and image is removed from Cloudinary

### PR DESCRIPTION
**Pull request by nikobatzi1990**
# Changes Made: added Admin GUI to manually delete products from database

## Issue No(s).:

Resolves #139 

# Before Submitting:
- [x] I have included a clear description of the contents of my Pull Request.
- [x] I have linked the associated Issue(s) above.
  - "Resolves #[Issue No.]"
  - Include a new line for each Issue if more than one is covered.
  - If there is no associated Issue, this PR will not be approved.
- [x] I have included the appropriate labels.
  - They should match the labels for the associated Issue(s).

# After Submitting:
- One of the Oriori team will review your code.
- Changes may be requested before approval, so please be sure to keep an eye on the status of your PR.

### CODE REVIEW

File: client/src/styles/AdminDeleteProduct.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+strong {
+  font-weight:bold;
+ }
```
Comment (kappanjoe) [inline client/src/styles/AdminDeleteProduct.css:14]: Sorry this is just a small nitpick, but could you adjust the spacing? There should be a space between `font-weight:` and `bold;`, and it also looks like there is a space before the closing `}` curly bracket.

File: client/src/pages/AdminDeleteProduct.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <ArrowSmallLeftIcon onClick={() => navigate('/home')}/>
+      </div>
+      
+      <div>
+        { products.length > 0
+        ? products.map((product) => (
+          <div key={ product.id } className="product-list-item" data-product-id={ product.id }>
+            <p><strong>Product ID:</strong> { product.id }</p>
+            <p><strong>Product Name:</strong> { product.product_name }</p>
+            <p><strong>Start Data:</strong> { product.start_date }</p>
```
Comment (kappanjoe) [inline client/src/pages/AdminDeleteProduct.tsx:51]: Small typo: Start **Data** should be Start **Date**

**Review by kappanjoe — CHANGES_REQUESTED** [review]
Everything looks great and it seems to work perfectly! Thank you so much for implementing this, it will help out a whole lot with managing products!! Just a couple quick code style fixes/typos if you don't mind fixing those~

File: client/src/styles/AdminDeleteProduct.css (hunk 1 above)
Comment (nikobatzi1990) [reply client/src/styles/AdminDeleteProduct.css]: Done!

File: client/src/pages/AdminDeleteProduct.tsx (hunk 2 above)
Comment (nikobatzi1990) [reply client/src/pages/AdminDeleteProduct.tsx]: And done!

**Review by kappanjoe — APPROVED** [review]
LGTM! 🥳

**Pull request merged**